moveit_planners/pilz_industrial_motion_planner/src/trajectory_generator.cpp

```diff
@@ -214,23 +214,25 @@ void TrajectoryGenerator::validateRequest(const planning_interface::MotionPlanRe
   checkGoalConstraints(req.goal_constraints, req.start_state.joint_state.name, req.group_name);
 }
 
-void TrajectoryGenerator::convertToRobotTrajectory(const trajectory_msgs::JointTrajectory& joint_trajectory,
+void TrajectoryGenerator::convertToRobotTrajectory(const planning_scene::PlanningSceneConstPtr& scene, 
```

Comment:
What about passing in the start state only? This makes the API more self-explanatory and more specific.

Reply:
Sure.